src/main/java/org/coffeemine/app/spring/components/HoursLogging.java

```diff
@@ -26,16 +34,42 @@ public HoursLogging() {
             fragment.setBegin(start_time.getValue());
             fragment.setEnd(end_time.getValue());
             // TODO: save selected task
+            final var id = db.addFragment(fragment);
+            task.addFragment(id);
+            Notification.show(
+                    "Task is now added!",
```

Comment:
Not a task, but a fragment. And anyway it should just show "sucess" or something like that